site2/website-next/blog/2022-12-01-pulsar-summit-asia-2022-recap.md

```diff
@@ -0,0 +1,108 @@
+---
+author: Sherlock Xu
+title: "Pulsar Summit Asia 2022 Recap"
+date: 2022-12-01
+---
+
+![](/img/pulsar-summit-asia-2022-recap-top.jpg)
+
+Since its inception in 2020, Pulsar Summit Asia has received increasing attention from both Asia and beyond. For Pulsar Summit Asia 2022, more than 1400 people from companies like Amazon, Tencent, IBM, Huawei, Dell, ByteDance, and Splunk registered for this online event to discuss the latest messaging and streaming technologies powering a wide variety of industries like education, food, gaming, e-commerce, and social media.
+
+Before we talk	about some of the Summit highlights, we would like to thank the entire Apache Pulsar community and all the friends from other open-source communities for making this event a great success! Additionally, this event would not have drawn the attention of such a broad audience without our speakers, Program Committee members, as well as community and media partners. Thank you all for your help and energy!
+
+<!--truncate-->
+
+And now, let’s look at some of the highlights and a round-up of this online virtual event:
+
+* 1400+ registrations and 40,000+ views globally
+* 41 speakers from companies like ByteDance, Huawei, Tencent, Nippon Telegraph and Telephone Corporation (NTT), Yum China, Netease, vivo, Nutanix, and StreamNative
+* 3 keynotes on Apache Pulsar and event-driven applications
+* 36 sessions on use cases, technical deep dives, and ecosystem talks
+* 14 Program Committee members
+* 18 community and media partners
+
+
+## Keynotes and sessions at a glance
+
+This two-day virtual event brought together engineers, architects, and data scientists from the messaging and streaming communities. They talked about Pulsar adoption for different use cases, event-driven platforms, technical details, and even Pulsar integration with other ecosystems. The following is a quick recap of some of the keynotes and sessions.
+
+
+### Keynotes
+
+* **A Cloud-Native, Unified Messaging and Streaming System for Modern Data Infrastructure (Mandarin)**: Jia Zhai, an Apache Pulsar PMC member, gave a high-level overview of Apache Pulsar and explained how it meets the requirements for messaging and streaming with its cloud-native features.
+* **What You Should Know about Apache Pulsar in 2022 (Mandarin)**: Penghui Li, an Apache Pulsar PMC member, talked about some of the existing problems in Pulsar and how the Pulsar community would work to solve them going forward.
+* **Event-Driven Applications Done Right (English)**: Matteo Merli, Apache Pulsar PMC Chair, provided his insights on the fundamentals of modern event-driven applications.
```

Comment:
Can we have the links to these keynotes?

Reply:
Ditto below talks.

Reply:
Do you mean the videos? These video links are not available right now. I believe @tuhaihe has more info.

Reply:
OK. Then let's leave it as is and update when they're aviable.